Automattic/wp-calypso
# Staging Sites: Disable SidebarNotice in the left menu

Related to https://github.com/Automattic/dotcom-forge/issues/1937

## Proposed Changes

In this PR, I propose to hide the sidebar upsell component for staging sites.

|Before|After|
|---|---|
| <img width="294" alt="Screenshot 2023-03-09 at 11 27 52" src="https://user-images.githubusercontent.com/727413/223997686-83533b42-575f-4c3e-910e-97dd8dd6bea8.png"> | <img width="289" alt="Screenshot 2023-03-09 at 11 31 28" src="https://user-images.githubusercontent.com/727413/223997683-dc10ce3c-6dd9-4ee8-8fa8-6b84ad8375e6.png"> |

## Testing Instructions



1. Create staging site
2. Confirm that upsell is not displayed in the sidebar
3. Confirm that upsell is still displayed on the non-staging site

## Pre-merge Checklist
